nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/StandardProcessorNode.java

```diff
@@ -187,6 +187,50 @@ public StandardProcessorNode(final Processor processor, final String uuid,
         }
 
         schedulingStrategy = SchedulingStrategy.TIMER_DRIVEN;
+        try
+        {
+
+            if(procClass.isAnnotationPresent(DefaultSchedule.class))
+            {
+                DefaultSchedule dsc = procClass.getAnnotation(DefaultSchedule.class);
+                try
+                {
+                    this.setSchedulingStrategy(dsc.Strategy());
+                }
+                catch (Throwable ex)
+                {
+                    LOG.error(String.format("Error while setting scheduling strategy from DefaultSchedule annotation: %s",ex.getMessage()));
```

Comment:
When logging exceptions, you could add the exception as second argument of the logging method, this will display the full stacktrace in the logs and that can be quite useful.
````java
LOG.error(String.format("Error while setting scheduling strategy from DefaultSchedule annotation: %s",ex.getMessage()), ex);
````